README.md

```diff
@@ -1,3 +1,4 @@
+[![Python 2.7 3.7](https://img.shields.io/badge/python-2.6%20%7C%202.7%20%7C%203.7-blue.svg)](https://www.python.org/)

```

Comment:
Missing 2.6 in the second pair of `[]`. This is for the tooltip IIRC.

Reply:
So I removed the 2.6 because I have no way of testing it at the moment, and our automated tests don't check py 2.6.
Maybe one for in discussion, I spent a short while trying to figure out how to install py 2.6, but I'm not sure if it's worth the time?